lib/src/material/material_desktop_controls.dart

```diff
@@ -31,8 +31,7 @@ class MaterialDesktopControls extends StatefulWidget {
   }
 }
 
-class _MaterialDesktopControlsState extends State<MaterialDesktopControls>
-    with SingleTickerProviderStateMixin {
+class _MaterialDesktopControlsState extends State<MaterialDesktopControls> with SingleTickerProviderStateMixin {
```

Comment:
You may want to revert the formatting changes, so that the CI doesn't complain when it attempts to format it.

That and it'll make it easier to see the actual change made.

Reply:
done

Reply:
Looks like there's still some formatting issues that CI is complaining about.

To fix those, downgrade to 3.27, then run dart formal in the codebase. Thanks in advance.

Reply:
done